data/thunderbirdceg.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "Gokulnath Panneerselvam",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Software developer with Cloud knowledge looking for remote roles",
+  "avatar": "https://github.com/thunderbirdceg.png",
+  "links": [
+    {
+      "name": "Follow me on Linkedin",
+      "url": "gokulnath-panneerselvam-20062619",
```

Unfortunately, the LinkedIn url you put here doesn't link to any profile. Add the link to your LinkedIn profile
```suggestion
      "url": "<paste in valid LinkedIn link here>",
```

---

I changed the linkedin url in my repo.
Is it ok now